### ISSUE TITLE
Weapons not automatically being added

### ISSUE THREAD

**Issue by TerraDK** [evil-morfar/RCLootCouncil2#189]
Ny'alotha raid weapon drops are not being picked up by RCLC automatically. We've noticed this going on for about 3-4 weeks. It happens in both Heroic and Mythic mode.

On Mythic BoD and TEP, it works fine.

I'd appreciate if you could take a look or let me know what you need from us. Thank you.

**Comment by evil-morfar** [comment]
Thanks for your report.
I'd like to have a look at your saved variables file located at `.../World of Warcraft/_retail_/WTF/Account/"account_name"/SavedVariables/RCLootCouncil.lua`.

**Comment by nebelhorn91** [comment]
We have the same problem since a few weeks. I tried to reset my settings and re-configure it, i uninstalled the addon, i installed an older version but still the same problem.

[RCLootCouncil.txt](https://github.com/evil-morfar/RCLootCouncil2/files/4798753/RCLootCouncil.txt)

I attached the file u asked for, since i cant upload .lua i renamed it to .txt, i hope this is okay.

Thanks for ur help :)
